test/e2e/lib/resources/constants.go

```diff
@@ -53,3 +55,9 @@ const (
 const (
 	KServiceKind string = "Service"
 )
+
+// WaitDeletionTime for deleting resources
+const (
+	WaitDeletionTime  = 20 * time.Second
```

Comment:
sounds good, will put this one to 10, and then in the few places you need 20, just multiply?

Reply:
I see you went with 20 everywhere. My only worry is that tests will last even longer... But I'm ok if you leave it as is